tests/functional/disaster-recovery/metro-dr-pc/test_failover_and_relocate.py

```
@@ -0,0 +1,344 @@
+                wl.workload_namespace,
+            )
+
+        # Verify the failover status from UI
+        if config.RUN.get("mdr_failover_via_ui"):
+            config.switch_acm_ctx()
+            verify_failover_relocate_status_ui(acm_obj)
+
+        # Start nodes if primary cluster was stopped
+        if primary_cluster_down:
+            logger.info(
+                f"Waiting for {wait_time} seconds before starting nodes of previous primary cluster"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)